api/filters.go

```diff
@@ -122,6 +124,10 @@ func (api *API) createFilter(w http.ResponseWriter, r *http.Request) {
 		DisclosureControl: nil, // populate for these fields yet
 	}
 
+	if v.IsBasedOn.Type == "cantabular_multivariate_table" {
```

Comment:
You're using a const and a non-const for the same value.. Above I mentioned the filter type should probably be changed. You should also add a `cantabularMultivariateTable` const and use that everywhere you need too.